MonMaraicher/Features/FarmerDetail/FarmerDetailsViewModel.swift

```diff
@@ -11,28 +11,51 @@ struct FarmerDetailsViewModel: Identifiable, Hashable {
     let id: UUID
     let title: String
     let phoneNumber: String?
+    let email: String?
     let products: [Products]
+    let websites: [Websites]
     let coordinate: CLLocationCoordinate2D
-    let markerSystemImageName: String
-    let directionButtonTitle: String
     let address: String
     let farmerAddressesTypes: [String]
     let city: String
 
+    let markerSystemImageName: String
+    let directionButtonImageSystemName: String
+    let phoneButtonImageSystemName: String
+    let emailButtonImageSystemName: String
+
+    var phoneCallURL: URL? {
+        guard let phoneNumber, let url = URL(string: "tel:\(phoneNumber)") else { return nil }
```

Comment:
if you type is URL? why do you check `URL(string: "tel:\(phoneNumber)")` ?
why not directly returning 
```
return URL(string: "tel:\(phoneNumber)")
```